src/OrchardCore.Modules/OrchardCore.Users/Views/Items/RegisterUserTask.Fields.Edit.cshtml

```diff
@@ -11,14 +11,14 @@
 
 <div class="form-group" asp-validation-class-for="ConfirmationEmailSubject">
     <label asp-for="ConfirmationEmailSubject">@T["Subject for confirmation email"]</label>
-    <input type="text" asp-for="ConfirmationEmailSubject" class="form-control code" />
+    <input type="text" asp-for="ConfirmationEmailSubject" class="form-control code" required />
```

Comment:
Doesn't it make it required by default, thought the Send Email is not checked by default? (or is it checked)

Reply:
I think it's let me double check ..

Reply:
![Register User Task](https://user-images.githubusercontent.com/3237266/129258455-6ff4ff94-0795-4186-88de-999c691984ea.png)

Reply:
That will only work on Create not on Edit, where the value may be different to the default js you are suggesting.

I would drop the javascript validation personally

Reply:
> That will only work on Create not on Edit

How?

> I would drop the javascript validation personally

We could remove the client-side validation and rely on server-side instead

Reply:
Because your js code assumes the box is checked. When editing the box maybe unchecked and the js code still makes it required.

Reply:
I remove the client-side validation and rely on server-side once